extra/html-extra/HtmlExtension.php

```diff
@@ -37,6 +37,7 @@ public function getFunctions(): array
     {
         return [
             new TwigFunction('html_classes', [self::class, 'htmlClasses']),
+            new TwigFunction('html_cva', [$this, 'cva']),
```

Comment:
Would it make sense to name it just `cva` (as I suppose `cva` only works in the context of an HTML doc anyway)?

Reply:
Make more sense I changed it 👍